openapi/generator.go

```diff
@@ -758,6 +758,14 @@ func (g *Generator) newSchemaFromStructField(sf reflect.StructField, required bo
 	if t, ok := sf.Tag.Lookup(formatTag); ok {
 		schema.Format = t
 	}
+
+	// Set example value from tag to schema
+	if e := sf.Tag.Get("example"); e != "" {
+		if parsed, err := parseExampleValue(schema.Type, e); err == nil {
```

Comment:
The error should be dealt-with if non-nil.
If suggest that we silently ignore the error but save it with`g.error(&FieldError{...`. See some examples in `generator.go`. Those errors can be retrived with `fizz.Errors`.